nesc/whip6/platforms/parts/mcu/cc26xx/uart/HalUARTAsyncWritePrv.nc

```diff
@@ -64,4 +65,6 @@ implementation {
             signal AsyncWrite.writeDone(SUCCESS);
         }
     }
+
+    default async event void AsyncWrite.writeDone(error_t result) { }
```

Comment:
This module provides only a single interface. Why do you want a default event handler?